src/pages/gen2/build-a-backend/data/customize-authz/grant-lambda-function-access-to-api/index.mdx

```diff
@@ -55,7 +55,7 @@ const schema = a
   })
   // highlight-start
   .authorization([
-    a.allow.resource(functionWithDataAccess).to(['query', 'listen'])
+    allow.resource(functionWithDataAccess).to(['query', 'listen'])
```

Comment:
should this be `.authorization(allow => [allow.resource(functionWithDataAccess).to(['query', 'listen'])])`?